src/main/java/com/arkvis/orchid/OvulationPredictor.java

```diff
@@ -1,13 +1,34 @@
 package com.arkvis.orchid;
 
 import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
 
 class OvulationPredictor {
 
     static final int DEFAULT_DAYS_BEFORE_PERIOD = 14;
 
-    LocalDate predictNextOvulationDate(PeriodWindow periodWindow) {
-        if (periodWindow.isEmpty()) return null;
-        return periodWindow.getStartDate().minusDays(DEFAULT_DAYS_BEFORE_PERIOD);
+    static final int DEFAULT_DAYS_BEFORE_OVULATION = 5;
+
+    private final List<LocalDate> nextFertilityWindow;
```

Comment:
We shouldn't have this as an instance variable because depending on the given dates it could change